webui/react/src/pages/DesignKit.tsx

```diff
@@ -930,6 +932,116 @@ const CodeEditorSection: React.FC = () => {
   );
 };
 
+const InlineFormSection: React.FC = () => {
+  const [inputValue, setInputValue] = useState('test value');
+  const [inputWithValidatorValue, setInputWithValidatorValue] = useState('');
+  const [textAreaValue, setTextAreaValue] = useState(
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Id tempora harum animi, provident explicabo in esse, hic qui cumque maiores ipsa, praesentium ad repellat dolorem voluptates! Eligendi magnam blanditiis velit.',
+  );
+  const [passwordInputValue, setPasswordInputValue] = useState('123456789');
+  const [inputSearchValue, setInputSearchValue] = useState('Some fancy search');
+  const [inputNumberValue, setInputNumberValue] = useState(1234);
+  const [selectValue, setSelectValue] = useState('off');
+
+  const inputCallback = useCallback((newValue: string) => {
+    setInputValue(newValue);
+  }, []);
+  const inputWithValidatorCallback = useCallback((newValue: string) => {
+    setInputWithValidatorValue(newValue);
+  }, []);
+  const textAreaCallback = useCallback((newValue: string) => {
+    setTextAreaValue(newValue);
+  }, []);
+  const passwordInputCallback = useCallback((newValue: string) => {
+    setPasswordInputValue(newValue);
+  }, []);
+  const InputNumberCallback = useCallback((newValue: number) => {
+    setInputNumberValue(newValue);
+  }, []);
+  const InputSearchCallback = useCallback((newValue: string) => {
+    setInputSearchValue(newValue);
+  }, []);
+  const selectCallback = useCallback((newValue: string) => {
+    setSelectValue(newValue === '1' ? 'off' : 'on');
+  }, []);
+
+  return (
+    <ComponentSection id="InlineForm" title="InlineForm">
+      <AntDCard>
+        <p>
+          The <code>{'<InlineForm>'}</code> allows people to have a simple form with just one input
+          to interact with.
+        </p>
+      </AntDCard>
+      <AntDCard title="Usage">
+        <p>
+          The <code>{'inputElement'}</code> sets the actual input to be rendered in the form.
+        </p>
+        <p>
+          If using the <code>{'Input.Password'}</code> component, is important to pass the{' '}
+          <code>{'isPassword'}</code> prop.
+        </p>
+        <br />
+        <div style={{ maxWidth: '700px' }}>
+          <InlineForm<string> initialValue={inputValue} label="Input" onSubmit={inputCallback}>
+            <Input />
+          </InlineForm>
+          <hr />
+          <InlineForm<string>
```

Comment:
Please split this into controlled and uncontrolled examples. Controlled examples should set `value` and `onSubmit` with a `useState` and uncontrolled should just set `initialValue` and then do nothing in the `onSubmit`.